1.12/Dockerfile

```diff
@@ -9,10 +9,14 @@ RUN set -xe \
 	&& ELIXIR_DOWNLOAD_SHA256="96167d614b9c483efc54bd7898c3eea4768569a77dd8892ada85d7800d5e3ea4" \
 	&& curl -fSL -o elixir-src.tar.gz $ELIXIR_DOWNLOAD_URL \
 	&& echo "$ELIXIR_DOWNLOAD_SHA256  elixir-src.tar.gz" | sha256sum -c - \
-	&& mkdir -p /usr/local/src/elixir \
-	&& tar -xzC /usr/local/src/elixir --strip-components=1 -f elixir-src.tar.gz \
+	&& mkdir -p /tmp/elixir /usr/local/src/elixir \
+	&& tar -xzC /tmp/elixir --strip-components=1 -f elixir-src.tar.gz \
 	&& rm elixir-src.tar.gz \
-	&& cd /usr/local/src/elixir \
-	&& make install clean
+	&& cd /tmp/elixir \
+	&& make install clean \
+	&& find /tmp/elixir \( -name "test" -o -name "examples" -o -name "pages" \) -type d -exec rm -rf {} + \
+	&& cp -r /tmp/elixir/lib /usr/local/src/elixir/ \
+	&& cp -r /tmp/elixir/.formatter.exs /usr/local/src/elixir/ \
```

Comment:
This is still maintaining the unicode directory, which is arguably the biggest one. Here is the command we use when creating the precompiled archive:

    zip -9 -r Precompiled-v$(VERSION).zip bin CHANGELOG.md lib/*/ebin lib/*/lib LICENSE man NOTICE README.md VERSION

Perhaps we should just copy `lib/*/lib`?

Reply:
Thank you, I've changed this step to just remove all the folders and files except matching `lib/*/lib` pattern.